branch_tags/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db                 import models
+from django.db.models.deletion import CASCADE
+
+from users.models              import User
+from postings.models           import Posting
+from keywords.models           import Keyword
+
+class User_tag(models.Model):
```

Comment:
class 명은 camelCase로 작성해주세요!
```suggestion
class UserTag(models.Model):
```